wgpu-types/src/lib.rs

```diff
@@ -1198,18 +1206,18 @@ impl Default for Limits {
             max_storage_buffers_per_shader_stage: 8,
             max_storage_textures_per_shader_stage: 4,
             max_uniform_buffers_per_shader_stage: 12,
-            max_uniform_buffer_binding_size: 64 << 10,
-            max_storage_buffer_binding_size: 128 << 20,
+            max_uniform_buffer_binding_size: 64 << 10, // (64 KiB)
+            max_storage_buffer_binding_size: 128 << 20, // (128 MiB)
             max_vertex_buffers: 8,
-            max_buffer_size: 256 << 20,
+            max_buffer_size: 256 << 20, // (256 MiB)
             max_vertex_attributes: 16,
             max_vertex_buffer_array_stride: 2048,
             min_uniform_buffer_offset_alignment: 256,
             min_storage_buffer_offset_alignment: 256,
             max_inter_stage_shader_components: 60,
             max_color_attachments: 8,
             max_color_attachment_bytes_per_sample: 32,
-            max_compute_workgroup_storage_size: 16384,
+            max_compute_workgroup_storage_size: 16352,
```

Comment:
This is unfortunately not the limit for WebGPU - https://gpuweb.github.io/gpuweb/#dom-supported-limits-maxcomputeworkgroupstoragesize

Reply:
yes, i saw that. if you look at older versions of the spec it does have that value however- it was changed. downlevel_defaults has 16352 and the docs claim also claim 16352. Higher is better, so lowering this is fine. If you want all of them changed to 16384 and ignore the apple requirement im fine with that too, but it has to be documented

Reply:
I think downlevel (i.e. old macs) having 16352 is fine, and non-downlevel (aka webgpu compliant) at 16384 is the right way to go. WebGPU has already decided that it doesn't mind losing those devices.

Reply:
Okay, I'll update the docs then!